README.md

```diff
@@ -31,4 +31,5 @@ Specifically:
 * `picknik_ur_site_config` extends the base configuration with capabilities for robots with physics and perception.
 * `picknik_ur_gazebo_config` extends the site configuration with support for the Gazebo simulator.
 * `picknik_ur_gazebo_scan_and_plan_config` extends the Gazebo configuration with an alternate environment for 3D object scanning.
+* `picknik_dual_ur_arm_config` is an example configuration for using multiple arms.
```

Comment:
Should we rename it to `picknik_ur_multi_arm_config` both to remove "dual" and for consistency with the other packages (ur first?)